jaxrs/src/main/java/com/cerner/beadledom/jaxrs/exceptionmappers/FailureExceptionMapper.java

```diff
@@ -0,0 +1,70 @@
+package com.cerner.beadledom.jaxrs.exceptionmappers;
+
+import static javax.ws.rs.core.Response.Status.INTERNAL_SERVER_ERROR;
+
+import com.cerner.beadledom.jaxrs.models.JsonError;
+
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+import javax.ws.rs.ext.Provider;
+
+import org.jboss.resteasy.spi.Failure;
+import org.jboss.resteasy.spi.ReaderException;
+import org.jboss.resteasy.spi.WriterException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Exception Mapper for all unhandled Exceptions extending from the {@link Failure} hierarchy
+ * (specifically for Resteasy Built-in Internally-Thrown Exceptions like {@link ReaderException},
+ * {@link WriterException}, etc.) thrown by the service. See
+ * http://docs.jboss.org/resteasy/docs/3.1.4.Final/userguide/html_single/index.html#builtinException
+ * for more information.
+ *
+ * @author Cal Fisher
+ */
+@Provider
+public class FailureExceptionMapper implements ExceptionMapper<Failure> {
+
+  private static final Logger logger = LoggerFactory.getLogger(FailureExceptionMapper.class);
+
+  /**
+   * Generates a Response object with a Status of the Failure, Content-Type 'application/json', and
```

Comment:
Same comment as above, single summary sentence, than continue documentation in a new paragraph.

Reply:
https://github.com/cerner/beadledom/pull/45/commits/865a017a55ce93a2c50c539260b17f6bbcdfe110#diff-94f0fc074c60a4a4d74f16bebf75afc9R34